components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3373,5 +3373,13 @@ void RewardsServiceImpl::StartProcess(StartProcessCallback callback) {
   StartLedger(std::move(callback));
 }
 
+void RewardsServiceImpl::GetWalletPassphrase(
+    const GetWalletPassphraseCallback& callback) {
+  if (!Connected()) {
+    return;
```

Comment:
Do we need to call the callback before we return here? Are the callers expecting a response?

Reply:
it's not expecting, but let me add it for consistency

Reply:
done